### ISSUE TITLE
Home assistant 107.6 config is not an object

### ISSUE THREAD

**Issue by gbdavidx** [thomasloven/lovelace-layout-card#62]
In between two cards i have a break, on the front end i have a red box that says config is not an object

```
resources:
  - url: /local/custom-lovelace/upcoming-media-card/upcoming-media-card.js
    type: js
  - url: /local/layout-card.js
    type: module


views:
  - type: custom:layout-card
    layout: vertical
    cards:
        - type: thermostat
          entity: climate.main_floor
        - break
        - type: 'custom:upcoming-media-card'
          entity: sensor.plex_recently_added
          title: Recently Downloaded
          image_style: fanart
```
https://imgur.com/rRiRqDh

**Comment by thomasloven** [comment]
Should be fixed in release 14.

**Comment by v-sante** [comment]
I'm still getting this problem on release 16 just installed.  Trying to put a break into vertical layout between two stack cards.  Works but get this exact same red box with config is not an object break.

**Comment by jffz** [comment]
Having the same issue with HA 0.114.3 and release 16.

**Comment by olegosipenko87** [comment]
HA 0.115 
problem still exists

**Comment by Rainerxyz** [comment]
Same problem with HA 0.117.6

**Comment by mmverk** [comment]
I had the same issue, it drove me mad. I had an existing lovelace dashboard that I tweaked to use the layout-card. That worked fine, the layouts and column widths got adjusted so the layout card seemed to work perfectly. Untill I used the - break... Same issue, the red rectangle with ' config is not an object'. My hunch was that it is not so much a bug in the layout card but somehow the combination of the already existing script entries. I tried numerous tweaks and none of them worked. 

In the end I removed the full existing script and started one from scratch. Of course I saved my existing script first as a seperate textfile. I configured the new script via the UI, simply with a few markdowns. I again edited that script to get it to use the layout card. And, I used the break command and it straight away worked. That script will be the base for the development of my lovelace dashboard.

To help you all that are still running into this issue I have copied my very simple dashboard script below that uses the layout card and has a working - break command.

views:
  - title: Hoofd
    path: hoofd
    badges: []
    cards:
      - type: 'custom:layout-card'
        layout: vertical
        column_num: 7
        cards:
          - type: markdown
            content: 'Test 1'
          - type: markdown
            content: 'Test 2'
          - type: markdown
            content: 'Test 3'
          - break
          - type: markdown
            content: 'Test 4'
          - type: markdown
            content: 'Test 5'
 
If you copy the above in a new dashboard I am confident it will also work for you.

Michel
